samples/net/dhcpv4_client/src/main.c

```diff
@@ -69,6 +83,11 @@ int main(void)
 
 	iface = net_if_get_default();
 
+	net_dhcpv4_init_option_callback(&dhcp_cb, option_handler,
+					42, ntp_server, sizeof(ntp_server));
```

Comment:
Please define a symbol for the magic number (42).

Reply:
Will do